pages/[lng]/login/index.tsx

```diff
@@ -40,9 +41,20 @@ const LoginPage: FC<any> = ({
   hasFacebookAuth
 }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
   const i18n: any = useI18n()
+  const recaptchaRef = useRef<any>()
 
   const linksBreadcrumb = [`${i18n.t("header.home")}`, `${i18n.t("login.title")}`]
```

Comment:
```suggestion
  const linksBreadcrumb = [i18n.t("header.home"), i18n.t("login.title")]
```

begitu juga bisa kan ya?